x-pack/plugins/fleet/server/services/epm/archive/parse.ts

```diff
@@ -558,6 +558,12 @@ export function parseDataStreamElasticsearchEntry(
     );
   }
 
+  if (expandedElasticsearch?.index_template?.data_stream) {
```

Comment:
@kpollich there are unit tests for this function if you could add a test case https://github.com/elastic/kibana/blob/2a9e7734e5e05caa60890a4e1b01c38c163a4be8/x-pack/plugins/fleet/server/services/epm/archive/parse.test.ts#L46

Reply:
Good call! Added some test cases here.